### DISCUSSION TITLE
Offline/Airgapped environment support in the community version

### DISCUSSION THREAD

**Discussion by IvanDrag0** [Budibase/budibase#12010] [Help] [answered]
My main use case to use Budibase involves using it in a disconnected/airgapped environment. I've looked through the Budibase website and documentation and noticed that the [Pricing](https://budibase.com/pricing/) page shows that the Enterprise version supports air-gapped deployment. The documentation doesn't really mention offline/airgapped deployment, however, I found a few issues that mention it (#3164, #10977, #3162, and several others). I also noticed that the docker-compose script references the `${OFFLINE_MODE}` variable, which is not found it in .env file. 

Based on those issues/documentation, I have a few questions:

- Does the current version of Budibase support offline/disconnected environments?
- If it does support it, is this functionality only found in the Enterprise version or also in the Community version?
- Is there any documentation out there regarding the requirements and procedures to deploy an airgapped version (to make sure that my environment is ready for it)?

**Comment by ConorWebb96** [comment, marked as answer]
Hey @IvanDrag0,

Yes these are enterprise tier features.

- For further information and assistance with your specific requirements, I recommend reaching out to our sales team through this contact [link](https://budibase.com/contact/). They will be happy to discuss your needs in more detail.

I hope this helps!
